@xen-orchestra/backups/_cleanVm.js

```diff
@@ -72,6 +72,30 @@ const mergeVhdChain = limitConcurrency(1)(async function mergeVhdChain(chain, {
 
 const noop = Function.prototype
 
+const listVhds = async (handler, vmDir) => {
```

Comment:
File structure can be found in https://github.com/vatesfr/xen-orchestra/blob/cb37f85d8e135b3fc939ee57a162600c9c1b868b/packages/xo-server/src/xo-mixins/backups-ng/index.js#L125-L139